qiskit/algorithms/minimum_eigen_solvers/adaptvqe.py

```diff
@@ -0,0 +1,332 @@
+# This code is part of Qiskit.
```

Comment:
This file can be removed as we only need the version above where `AdaptVQE` contains a `VQE` instead of deriving from it.